src/app/modules/group/components/member-list/member-list.component.ts

```diff
@@ -58,20 +57,8 @@ const descendantTeamsColumns: Column[] = [
 
 const membersLimit = 25;
 
-interface Data {
-  columns: Column[],
-  rowData: (
-    (Member|GroupChild|{ login: string, parentGroups: string }|{ name: string, parentGroups: string, members: string }) &
-    { route: RawGroupRoute }
-  )[],
-}
-
-interface DataFetching {
-  route: GroupRoute,
-  filter: Filter,
-  sort: string[],
-  fromId?: string,
-}
+type Row = (Member|GroupChild|{ login: string, parentGroups: string }|{ name: string, parentGroups: string, members: string }) &
```

Comment:
It it not related with your change but I don't like at all this type... rows are not mixed, it is just one type depending of the tab, there many ways to handle that more properly.